src/components/app/details/triggerView/workflow/Workflow.tsx

```diff
@@ -263,19 +271,14 @@ export class Workflow extends Component<WorkflowProps> {
             >
                 <div className="workflow__header">
                     {this.props.fromAppGrouping ? (
-                        <>
-                            <input
-                                type="checkbox"
-                                className="mt-0-imp cursor icon-dim-16"
-                                data-app-id={this.props.appId}
-                                checked={this.props.isSelected}
-                                onChange={this.props.handleSelectionChange}
-                                id={`chkValidate-${this.props.appId}`}
-                            />
-                            <label className="ml-12 cursor fs-13 mb-0-imp lh-20" htmlFor={`chkValidate-${this.props.appId}`}>
-                                {this.props.name}
-                            </label>
-                        </>
+                        <Checkbox
+                            rootClassName="fs-13 fw-6 mb-0 app-group-checkbox"
+                            isChecked={this.props.isSelected}
+                            value="CHECKED"
```

Comment:
Can we reuse constant for "CHECKED" here

Reply:
Done